ISISComputingGroup/ibex_gui
# Ticket2587: E4 Nicos Scripting

### Description of work

Converted the NICOS scripting perspective to E4.

### To test

* The perspective can be opened in e4
* The view scales properly, and scroll bars appear at appropriate window sizes

### Acceptance criteria

*List the acceptance criteria for the PR*

---

#### Code Review

    - Manual system tests for the functionality should be added if there are no automated tests
    - Manual system tests can be removed from the template if they are covered by suitable automated tests

### Functional Tests


### Final Steps
